include/zephyr/bluetooth/audio/has.h

```diff
@@ -213,6 +214,46 @@ int bt_has_client_conn_get(const struct bt_has *has, struct bt_conn **conn);
  */
 int bt_has_client_presets_read(struct bt_has *has, uint8_t index, uint8_t max_count);
 
+/**
+ * @brief Switch Active Preset.
+ *
+ * Client method to set preset identified by @p index as active.
+ * The status is returned in the @ref bt_has_client_cb.preset_switch callback.
+ *
+ * @param has Pointer to the Hearing Access Service object.
+ * @param index Preset index to activate.
+ * @param sync Request active preset synchronization in set.
+ *
+ * @return 0 in case of success or negative value in case of error.
+ */
+int bt_has_client_preset_switch(struct bt_has *has, uint8_t index, bool sync);
```

Comment:
I wonder about the choice to call this "switch" rather than "set" (especially since the spec uses the term "set"). 

I guess "switch" implies that if the active preset is index 5, and the index passed to this is also 5, then it is a no-op - Is that the intention?

Reply:
I don't have that feeling, but maybe this can be considered like this.
I don't mind to change it to `_set`, but I choose `_switch`, because the callback is named `preset_switch`